textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcher.java

```diff
@@ -23,7 +23,7 @@
  * @author laishuying
  */
 public class RegexMatcher implements IOperator {
-    private final IPredicate predicate;
+    private final RegexPredicate regexPredicate;
```

Comment:
All our index-based operators requires an "iPredicate" object.  I understand we need "RegexPredicate" from the RE2J package.  Is it possible to wrap it as an "iPredicate" object?  @JavierJia and @sandeepreddy602 : thoughts?